Sources/Prometheus/PrometheusMetrics.swift

```diff
@@ -64,6 +64,37 @@ private class MetricsHistogram: RecorderHandler {
     }
 }
 
+/// This is a `swift-metrics` timer backed by a Prometheus' `PromHistogram` implementation.
+/// This is superior to `Summary` backed timer as `Summary` emits a set of quantiles, which will be impossible to correctly aggregate when one wants to render a percentile for a set of multiple instances.
+/// `Histogram` aggregation is possible with some fancy server-side math.
```

Comment:
```suggestion
```

While in general I think it's a good thing to add doc blocks to classes & structs, seeing this is an internal class going along with a handful of other ones without docs, I think we should omit them here too.